staging/src/k8s.io/apimachinery/pkg/conversion/unstructured/converter.go

```diff
@@ -57,18 +57,20 @@ var (
 	unmarshalerType        = reflect.TypeOf(new(encodingjson.Unmarshaler)).Elem()
 	mapStringInterfaceType = reflect.TypeOf(map[string]interface{}{})
 	fieldCache             = newFieldsCache()
+	Converter              = newConverter()
+
 )
 
-// Converter knows how to convert betweek runtime.Object and
+// ConverterImpl knows how to convert betweek runtime.Object and
 // Unstructured in both ways.
-type Converter struct {
+type ConverterImpl struct {
```

Comment:
Make this private, and the constructer public, and expose a public interface.  Basically hide impl and expose interface and constructor.